website2/docs/guides-python-topologies.md

```diff
@@ -181,8 +181,7 @@ $ heron submit local \
 Note the `-` in this submission command. If you define a topology by subclassing `TopologyBuilder` you do not need to instruct Heron where your main method is located.
 
 > #### Example topologies buildable as PEXs
-> * See [this repo](https://github.com/streamlio/pants-dev-environment) for an example of a Heron topology written in Python and deployable as a Pants-packaged PEX.
-> * See [this repo](https://github.com/streamlio/bazel-dev-environment) for an example of a Heron topology written in Python and deployable as a Bazel-packaged PEX.
+> TODO
 
 ## Defining a topology by subclassing the `Topology` class
```

Comment:
A quick scan through the diffs between the two files locally makes it look like [guides-python-topologies](https://heron.incubator.apache.org/docs/guides-python-topologies) is an old version of [topology-development-topology-api-python.md](https://heron.incubator.apache.org/docs/topology-development-topology-api-python), due to it using an incorrect/old way of interpolating the version, lack of hyperlinks, and diff (extra/duplicate? content at the bottom). Should the former be entirely replaced with the latter in the master+version markdowns?

Reply:
agreed and removed `guides-python-topologies`